backend/hitas/models/housing_company.py

```diff
@@ -143,14 +164,24 @@ def __str__(self):
 
 # Rakennus
 class Building(ExternalHitasModel):
+    housing_company = models.ForeignKey(HousingCompany, on_delete=models.PROTECT)
```

Comment:
Could you add `related_name` to all Foreign Keys?
This makes it more easy to find the relation attribute name, when its explicitly decleared.

Reply:
I would not add a relation to a HousingCompany from a Building, we already have the relation through RealEstate, so this would be duplicate data and add the chance for erroneous data if Building has a different HousingCompany than RealEstate

Reply:
> I would not add a relation to a HousingCompany from a Building, we already have the relation through RealEstate, so this would be duplicate data and add the chance for erroneous data if Building has a different HousingCompany than RealEstate

Yeah, I added it originally as the implementation was a bit different and I needed it but it's true it's unnecessary now. I'll remove it

Reply:
> Could you add `related_name` to all Foreign Keys?

Added `related_name` where it made sense.